builtin/game/item.lua

```diff
@@ -205,6 +207,16 @@ function core.get_node_drops(nodename, toolname)
 	return got_items
 end
 
+local non_player_name = "!nonplayer"
```

Comment:
Was mainly for protection stuff, so I didn't need to choose whether nil / non players should be always protected or never protected. Or should minetest.is_protected be required to accept nil also?